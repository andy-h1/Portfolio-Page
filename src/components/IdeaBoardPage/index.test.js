import React from 'react';
import { render } from '@testing-library/react';
import { IdeaBoardPage } from '.';

describe('Form Component', () => {
  it('should render', () => {
    const { container } = render(<IdeaBoardPage />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
