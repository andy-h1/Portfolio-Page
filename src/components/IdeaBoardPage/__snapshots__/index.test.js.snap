// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Component should render 1`] = `
<div>
  <h2>
    Idea Board
  </h2>
  <button
    type="button"
  >
    + Add List
  </button>
  <button
    type="button"
  >
    + Add card
  </button>
  <div>
    <h3>
      Title:
    </h3>
    <p>
      Description: 
    </p>
  </div>
  <div>
    <h3>
      Title:
    </h3>
    <p>
      Description: 
    </p>
  </div>
</div>
`;
